### ISSUE TITLE
AISpace2 on Docker Hub

### ISSUE THREAD

**Issue by darekjk** [AISpace2/AISpace2#187]
Hi,

I created a docker image with AISpace version 0.8.11 and published it on Docker Hub. Link to my repository: https://hub.docker.com/r/darekjk/aispace2. Image use ubuntu 20.04, python3.9 and node 14.

Best regards,
Dariusz Jankowski

**Comment by ZijiaZhang** [comment]
Thank you for the contribution. Would you mind I mention your image in my docker repo?  https://github.com/ZijiaZhang/AISpace2/wiki

**Comment by darekjk** [comment]
Yes, sure, but I have one issue to resolve - cannot display results. 

For example, code:

display(solver)
solver.solve_one()

give me result:
Con_solver(graph=CSP({'action_0': {'Goto4', 'DeopOff', 'PickUp', 'Goto3', 'Goto2', 'Goto1'}, 'action_1': {'Got…

I guess I should receive some picture. probably there is a problem with ipywidgets. Did any one installed AISpace with python 3.9 and node 14?

Thanks for any help.
